ui/src/utils/api.js

```diff
@@ -171,6 +172,7 @@ export default {
   getExplorerConfig: async (dir, type) => (await axios.get(urls.getExplorerConfig, { params: { dir, type } })).data,
   checkFileExists: async (mode, workspace, path) => (await axios.get(urls.checkFileExists, { params: { mode, workspace, path } })).data,
   exportFiles: async (payload) => (await axios.get(urls.exportFiles, { ...payload })).data,
+  getPdf: async (payload, folder) => (await axios.get(urls.getPdf(folder), { ...payload })).data,
```

Comment:
Why do you introduce a new function in api.js? You could use the current API for file downloads like with images. Can you check?

Reply:
Thank you for the review, I made the changes as we discussed.